rest-api-spec/src/main/resources/rest-api-spec/test/cat.thread_pool/10_basic.yml

```diff
@@ -13,7 +13,7 @@
 
   - do:
       cat.thread_pool:
-        thread_pool_patterns: search,search_throttled,index_searcher,generic
+        thread_pool_patterns: search,search_throttled,generic
```

Comment:
Thanks @jed326 for preparing the PR. Could you explain why we remove `index_searcher` here?

Reply:
2 reasons:
1. After removing the feature flag `index_searcher` will show up in the output below. We need to either remove this here or create a new test with the correct `version` skipping.
2. The purpose of this time is to check that the metric is -1 for an unsupported threadpools and not for a supported threadpool. IMO this test is more about checking the formatting as opposed to accuracy so I don't think we need to update this test every time we add support for this metric to a given threadpool since we would end up with many different versions of the same test.

That's why I went with removing `index_searcher` from the arguments here.

Reply:
@jed326 could you please split this test in 2, pre / post concurrent search, so we could be sure the pool is there, thank you

```
- skip:
      version: " - 2.99.99"
      reason:  deleted docs and concurrent search are added in 2.12.0
      features: node_selector
  - do:
      cat.shards:
        help: true
      node_selector:
        version: "3.0.0 - "
```

Reply:
Added test case `"Test cat thread_pool total_wait_time output with concurrent search thread_pool":` in the latest revision.